lib/secure_headers/headers/content_security_policy/source_expression.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module SecureHeaders
+  class ContentSecurityPolicy
+    class SourceExpression
```

Comment:
I wonder if we should just call this a "expression" even something more general, rather than a "source expression". Rather than making this sound like it's specifically for a certain subset of possible values, it would include report paths, as well as unquoted directives like `script-src`.